Example/Pods/PINCache/PINCache/PINCache.h

```diff
@@ -94,18 +94,47 @@ typedef void (^PINCacheObjectContainmentBlock)(BOOL containsObject);
  @param name The name of the cache.
  @result A new cache with the specified name.
  */
-- (instancetype)initWithName:(NSString *)name;
+- (instancetype)initWithName:(nonnull NSString *)name;
```

Comment:
Yeah, this is actually in PINCache and I my guess is someone was just trying to be consistent when they put in the nullable serializer / deserializer. I didn't catch there was a NS_ASSUME above.